docs/metrics/README.md

```diff
@@ -0,0 +1,61 @@
+# OpenTelemetry .NET Metrics
+
+## Best Practices
+
+- When emitting metrics with dimensions, ALWAYS provide the dimensions in an
```

Comment:
lets also add that "Instruments must be created once and re-used throughout the application.", and link to 2 ways to do that.
1. Static in the console app example.
2. DI using the asp.net core example.

Reply:
Could we change this to say, `ALWAYS provide dimensions in same order` rather than sorted order to keep it simple. As the overhead is only the first time we see a new combination.

Reply:
Address both of the above points.